README.md

```diff
@@ -1,14 +1,23 @@
 # Media Capture from DOM Elements
 
-Documents the use of `captureStream` and `captureStreamUntilEnded` to acquire the
-rendered output of media elements (`<video>` and `<audio>`).
+Documents the use of `captureStream()` to acquire the rendered output of media elements (`<video>` and `<audio>`).
 
 [Editor's Copy](https://w3c.github.io/mediacapture-fromelement)
 
-## License
 
-This work is currently licensed under a Creative Commons Attribution-NoDerivatives 4.0
-International (CC BY-ND 4.0) License. See
-http://creativecommons.org/licenses/by-nd/4.0/legalcode
+Notes on bikeshedding :bicyclist:
+--------------
 
-Copyright Cisco and Mozilla
+To compile `index.bs` into `index.html` , I'm using the online compiler:
+
+```
+curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F force=1 > index.html
+```
+
+if the produced file has a strange size (i.e. zero, a few KBs), then something went terribly wrong; run instead:
+
+```
+curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F output=err
+```
+
+and try to figure out why `bikeshed` did not like the `.bs` :'(
```

Comment:
Please just make a script that does this automatically.  The checks should all be automated.  Ideally, the default invocation of `make` would do this.